metaflow/plugins/argo/argo_workflows_cli.py

```diff
@@ -186,7 +186,7 @@ def argo_workflows(obj, name=None):
 )
 @click.option(
     "--enable-error-msg-capture/--no-enable-error-msg-capture",
-    default=False,
+    default=True,
```

Comment:
function defaults for this option down the code path could also be changed to keep things consistent